### DISCUSSION TITLE
N64 prototype/dev cart reading support

### DISCUSSION THREAD

**Discussion by muguemjee** [sanni/cartreader#196] [Ideas & Suggestions]
What would be the likelihood of the Sanni cart reader supporting the backup of N64 prototype ROMs in the future? I'm curious because it supports several systems and prototype carts for them, which is really great. It being able to dump N64 protos would be even more amazing.

**Comment by sanni** [comment]
Given the price of those proto carts it's highly unlikely that I'll ever be able to add support for it. But maybe someone else will do it someday.

**Reply by muguemjee** [reply]
It's apparently the lack of the PIF chip that makes it not possible to dump them. Is this the case? I've no idea the likelihood of coming by this component. But would it theoretically be as straight-forward as implementing this chip, if it were somehow made available, in order to make N64 protos able to be dumped with the reader? For example, if someone assembles one up to focus only on N64 (like some youtube vids demonstrate with just the slot for the Game Boy systems), if that makes it more practical.

**Reply by networkfusion** [reply]
Perhaps this would be of help for a pif https://github.com/jago85/UltraPIF_Hardware / https://github.com/jago85/PifChecksum

**Reply by muguemjee** [reply]
> Given the price of those proto carts it's highly unlikely that I'll ever be able to add support for it. But maybe someone else will do it someday.

Sanni, I spoke with someone who's close around your region and they're willing to lend you one of their N64 prototype cartridges if it would help with a hardware update to support those. What do you think about that?

**Comment by mariomadproductions** [comment]
More discussion here: https://github.com/sanni/cartreader/discussions/295
